### DISCUSSION TITLE
RE7 - RT version low FPS with 40ish% GPU usage, non-RT version 90+ FPS with similar GPU Usage

### DISCUSSION THREAD

**Discussion by joeygreco1985** [praydog/REFramework#936] [General Questions]
So last night I was trying to get RE7 going with my RTX 4090, i7 13700k, and Quest 3. When I play the RT version of RE7 with the mod installed my FPS is in the 40-50 range, and my GPU usage is rather low in the 40 range. Increasing or decreasing the render resolution and disabling RT has no effect, the framerate and GPU usage does not change. 

So I switched the game over to the non-RT version and installed the proper framework files for this version. I'm now over 90+ fps so theres no stutter, but my GPU usage is still in the same 40-50 range. I get the same result with OpenVR via Steam and with OpenXR via Oculus. Is there a bug or setting I'm missing to cause this?

Im using Windows 11 with HAGS disabled. Other VR games work with no issue, this is my first time trying the RE VR mods.

**Comment by joeygreco1985** [comment]
Just an update that I now have the same issue in the non-RT version. My FPS is in the 45-50 range with 20% GPU usage. Not sure what changed.

**Comment by joeygreco1985** [comment]
Another update, I rolled back to nvidia driver 537.58 as per a suggestion from the SteamVR Steam discussion forum, as well as install the latest SteamVR beta, and its working correctly again.

**Comment by joeygreco1985** [comment]
Another update (sorry), I uninstalled the non-RT version, reinstalled the RT version with the correct mod files, and the original problem is still persisting. So im back at square one again, not sure what else to try.

**Comment by davidflana** [comment]
I have the same issue.  I posted my story in another thread above this one.  It has to be some sort of update to steam VR because I am half way through RE 7 and now it is not playable....  My initial issue was fixed when I disabled Steam VR Theater mode by renaming the exe file (google it or see my other recent post).  Did you try that?

**Reply by joeygreco1985** [reply]
No I haven't, my copy of the Steam_desktop_game_theatre folder doesn't have the executable in it. Has it moved to another location?

**Reply by davidflana** [reply]
maybe....  now that you mention it, when I first launch the game, it starts up in a bent theater looking screen for a bit before it goes full screen....  time to look for that exe!

**Reply by davidflana** [reply]
can't find it....

**Comment by joeygreco1985** [comment]
Another update, I updated to Nvidia driver 546.29 and SteamVR Beta 2.1.9 today to do some more testing. I found that now when the game launches, it's still opening the game in a theatre window, but if I switch to the actual desktop and minimize the desktop window, then switch to the game itself in SteamVR it seems to be running normally again. But at the same time it doesn't look like its respecting my headset's resolution as the game does not look as sharp as other VR titles. But it's running smooth enough to play at least?

**Reply by davidflana** [reply]
interesting....  did you try that minimize stuff before you updated the driver and went to steamVR beta?

**Reply by joeygreco1985** [reply]
I did try it on previous versions of SteamVR and the nvidia driver but it didn't seem to do anything

**Reply by davidflana** [reply]
IT'S FIXED!  I just went on to mess with it.  I checked for a steam update, and there was one.  I applied the update, and now it works great again!  It still launches in a theater looking window to start, but it works great again!  Check for the update and see if it fixes it for you too.

**Reply by joeygreco1985** [reply]
which versions of Steam and SteamVR are you running?

**Reply by davidflana** [reply]
steam version 1701289036 (Nov 29) and SteamVR version 2.1.8 (1701324388 (Nov 30)
